packages/core/core/src/ReporterRunner.js

```diff
@@ -85,7 +86,16 @@ export default class ReporterRunner {
     this.report(event);
   };
 
-  async report(event: ReporterEvent) {
+  async report(unsanitisedEvent: ReporterEvent) {
+    let event: ReporterEvent = unsanitisedEvent;
+    if (event.diagnostics) {
```

Comment:
Is this really needed? Shouldn't the `anyToDiagnostic` fix this case?

Reply:
`anyToDiagnostic` is only used for build failures, not things that come from the logger. The thinking here was to ensure all diagnostics were valid before going out to the reporters.